docker/datahub-frontend/Dockerfile

```diff
@@ -8,46 +8,47 @@ ARG ALPINE_REPO_URL=http://dl-cdn.alpinelinux.org/alpine
 ARG GITHUB_REPO_URL=https://github.com
 ARG MAVEN_CENTRAL_REPO_URL=https://repo1.maven.org/maven2
 
-RUN addgroup -S datahub && adduser -S datahub -G datahub
+# Add user and set up repository mirrors
+RUN addgroup -S datahub && adduser -S datahub -G datahub \
+    && if [ "${ALPINE_REPO_URL}" != "http://dl-cdn.alpinelinux.org/alpine" ]; then \
+    sed -i "s#http.*://dl-cdn.alpinelinux.org/alpine#${ALPINE_REPO_URL}#g" /etc/apk/repositories; \
+    fi
 
-# Optionally set corporate mirror for apk
-RUN if [ "${ALPINE_REPO_URL}" != "http://dl-cdn.alpinelinux.org/alpine" ] ; then sed -i "s#http.*://dl-cdn.alpinelinux.org/alpine#${ALPINE_REPO_URL}#g" /etc/apk/repositories ; fi
+# Upgrade packages and install dependencies
+ENV JMX_VERSION=0.18.0 \
+    LD_LIBRARY_PATH="/lib:/lib64"
 
-# Upgrade Alpine and base packages
-# PFP-260: Upgrade Sqlite to >=3.28.0-r0 to fix https://security.snyk.io/vuln/SNYK-ALPINE39-SQLITE-449762
-ENV JMX_VERSION=0.18.0
 RUN apk --no-cache --update-cache --available upgrade \
     && apk --no-cache add curl sqlite libc6-compat snappy \
-    && apk --no-cache add openjdk17-jre-headless --repository=${ALPINE_REPO_URL}/edge/community \
-    && apk --no-cache add jattach --repository ${ALPINE_REPO_URL}/edge/community/ \
+    && apk --no-cache add openjdk17-jre-headless jattach --repository=${ALPINE_REPO_URL}/edge/community \
     && wget ${GITHUB_REPO_URL}/open-telemetry/opentelemetry-java-instrumentation/releases/download/v1.24.0/opentelemetry-javaagent.jar -O opentelemetry-javaagent.jar \
     && wget ${MAVEN_CENTRAL_REPO_URL}/io/prometheus/jmx/jmx_prometheus_javaagent/${JMX_VERSION}/jmx_prometheus_javaagent-${JMX_VERSION}.jar -O jmx_prometheus_javaagent.jar
 
-ENV LD_LIBRARY_PATH="/lib:/lib64"
-
 FROM base as prod-install
 
 COPY ./datahub-frontend.zip /
 RUN unzip datahub-frontend.zip -d /datahub-frontend \
-   && mv /datahub-frontend/main/* /datahub-frontend \
-   && rmdir /datahub-frontend/main \
-   && rm datahub-frontend.zip
+    && mv /datahub-frontend/main/* /datahub-frontend \
+    && rmdir /datahub-frontend/main \
+    && rm datahub-frontend.zip \
+    && chown -R datahub:datahub /datahub-frontend \
+    && chmod 755 /datahub-frontend
+
 COPY ./docker/monitoring/client-prometheus-config.yaml /datahub-frontend/
-RUN chown -R datahub:datahub /datahub-frontend && chmod 755 /datahub-frontend
```

Comment:
Might be an out of order issue here. If the chown/chmod was expected to apply to the config file, it is no longer. I'd move the COPY above